FetchingFriends/fetching-friends-ui
# Feature/favorite pet

## What does this PR do?
-Adds a favorite feature to our pet cards
Allows people to favorite any type of pet and save it to their dashboard

## How should it be tested in the UI?
- Click on a pet category click the heart on the pet you like
- Go home and watch that pet appear on the dashboard
- (Does not update the card instantly & Requires user to go home to see updated favorites)
